.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,19 @@
+name: formatter-shfmt
+on:
+  [push, pull_request]
+jobs:
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@v0.2.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SHFMT_OPTS: -i=2 -ln=bash -fn -ci -sr # arguments to shfmt.
+        with:
+          sh_checker_comment: true
+          sh_checker_exclude: "documentation images sounds etc"
+          sh_checker_shfmt_disable: false
+          sh_checker_shellcheck_disable: true
+          sh_checker_checkbashisms_enable: false
```

Comment:
Hmmm... could you explain how this thing works? I did not get where we installed shfmt. There is some magic behind it? Also, do you have an example in your fork?

Reply:
> Also, do you have an example in your fork?

The example I do have is for pushes, which would be different than when we have a PR (with PRs a bot will make a post similar to what codecov does)
https://github.com/alan-barzilay/kworkflow/actions